spec/models/business_contact_form_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe BusinessContactForm do
+  describe 'validation' do
+    it 'is not valid if the first name is missing' do
+      params = {
+        first_name: '',
+        last_name: 'Doe',
+        email: 'john_doe@example.com'
+      }
+      business_contact_form = described_class.new(params)
+
+      expect(business_contact_form).not_to be_valid
+    end
+
+    it 'is not valid if the last name is missing' do
+      params = {
+        first_name: 'John',
+        last_name: '',
+        email: 'john_doe@example.com'
+      }
+      business_contact_form = described_class.new(params)
+
+      expect(business_contact_form).not_to be_valid
+    end
+
+    it 'is not valid if the email address is missing' do
```

Comment:
This test makes obvious sense to me. I just wonder how much it bothers us when someone gives only their name and email address, or only their surname and email address. Maybe it's not that big of a problem. WYT?